### PR TITLE
PR nova printhead - beaugalbraith

### PR DESCRIPTION
**Pull request by beaugalbraith**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Nova printhead for Afterburner.
#### How was it tested? 
Tested on my v2.4
#### Any background context you want to provide?
Nobody had made one yet. I used the official cad to modify the mosquito/v6 printheads to suit the Nova hotend.
#### Screenshots (if appropriate)
In my folder

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ like so:
 |Badnoob |[Precision AB Drive](./Badnoob/Precision_AB_Drive)|Changes the gearing for the AB drives to improve both resolution and torque, with a strenthened frame. |:grey_question:|:grey_question:|:heavy_check_mark:|:x:|
 | |[AB-BN Afterburner mod](./Badnoob/AB-BN)|Improved Afterburner with 5015 fan |:x:|:heavy_check_mark:|:heavy_check_mark:|:grey_question:|
 | |[Rail Alignment](./Badnoob/Rail_Alignment)|Tools and procedure to properly align the rails in 2.4 and Afterburner|:x:|:x:|:heavy_check_mark:|:x:|
+|beaugalbraith| [Nova Printhead](./beaugalbraith/Nova_Printhead) | Use Nova Hotend with Afterburner | :grey_question: | :grey_question: | :heavy_check_mark: | :grey_question: | :grey_question:|
```
Comment (KiloQubit) [inline printer_mods/README.md:55]: Can you move your line down so it's in alphabetical order by name?  It should be after `Baltojikale`

File: printer_mods/beaugalbraith/Nova_Printhead/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Printhead for the Nova hotend. Only tested on my 2.4.
+
```
Comment (KiloQubit) [inline printer_mods/beaugalbraith/Nova_Printhead/README.md:2]: Can you include your images in the README?  The syntax is

    IMG_7030

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please see README fixes below.

File: printer_mods/README.md (hunk 1 above)
Comment (beaugalbraith) [reply printer_mods/README.md]: Yep, can't believe I messed that up. Fixed with commit d4f824a

File: printer_mods/beaugalbraith/Nova_Printhead/README.md (hunk 2 above)
Comment (beaugalbraith) [reply printer_mods/beaugalbraith/Nova_Printhead/README.md:3]: Fixed with commit b7ae011

**Pull request merged**